app/src/main/java/com/epfl/drawyourpath/path/RunsAdapter.kt

```diff
@@ -23,6 +26,20 @@ class RunsAdapter(private val runs: List<Run>) : RecyclerView.Adapter<RunsAdapte
     override fun onBindViewHolder(holder: ViewHolder, position: Int) {
         val run = runs[position]
 
+
+        val runCoordinates:List<LatLng> = run.getPath().getPoints() // Get the coordinates for this specific run
+        val apiKey = "AIzaSyCE8covSYZE_sOv4Z-HaoljRlNOTV8cKRk";

```

Comment:
The API key should not be visible, it is ok for now (given the private discussions we had).